### PR TITLE
Update filesize on the File after signing.

### PR DESCRIPTION
**Pull request by eviljeff**

### CODE REVIEW

**Comment by eviljeff** [comment]
fixes #1962 - the first part anyway.

File: src/olympia/lib/crypto/tests.py [hunk 1]
```diff
@@ -218,6 +219,13 @@ def test_is_signed(self):
         packaged.sign_file(self.file_, settings.SIGNING_SERVER)
         assert packaged.is_signed(self.file_.file_path)
 
+    def test_size_updated(self):
+        unsigned_size = storage.size(self.file_.file_path)
+        packaged.sign_file(self.file_, settings.SIGNING_SERVER)
+        signed_size = storage.size(self.file_.file_path)
+        assert self.file_.size == signed_size
+        assert unsigned_size != signed_size
```
Comment (wagnerand) [inline src/olympia/lib/crypto/tests.py]: We could do `<=`?

Comment (eviljeff) [inline src/olympia/lib/crypto/tests.py]: < ?

Comment (wagnerand) [inline src/olympia/lib/crypto/tests.py]: `<`. I wrote `<=` first but that obviously defeats the purpose.

**Comment by EnTeQuAk** [comment]
r+wc (from @wagnerand)

**Pull request merged**